ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -159,7 +159,7 @@ func validateCommaSeparatedParam(cfnParams *cloudformation.CfnStackParams, param
 	return false
 }
 
-func createCluster(context *cli.Context, ecsClient ecsclient.ECSClient, cfnClient cloudformation.CloudformationClient, cliParams *config.CLIParams) error {
+func createCluster(context *cli.Context, ecsClient ecsclient.ECSClient, cfnClient cloudformation.CloudformationClient, ssmClient ssm.Client, cliParams *config.CLIParams) error {
```

Comment:
No op: This is fine, but note for future self to maybe consider converting some of the arguments here into a parameter object so we don't have quite so many in the method signature.